docs/cip/README.md

```diff
@@ -4,704 +4,2873 @@ Title: Ouroboros Leios - Greater transaction throughput
+benefits once sustained throughput exceeds 50-70 TxkB/s (36-50 TPS
+equivalent)</em>
+
+</div>
+
+Achieving this capacity increase requires trade-offs, as detailed below.
+
+<a name="time-to-market"></a>**2. Reasonable time to market: Complexity
+trade-offs**
+
+The linearization approach avoids complex distributed systems problems around
+transaction sharding and sophisticated mempool coordination that could delay
+deployment by years. Critically, this linearization approach also simplifies
+conflict resolution compared to highly concurrent variants. In the proposed
+protocol, invalid transactions typically arise only in circumstances where most
+honest stake immediately recognizes them as invalid and can treat them as such
+without ambiguity. In contrast, highly concurrent variants like Input
+Block-based protocols or more generally protocol with additional decoupled block
+production create scenarios where honest nodes become helpless when conflicts
+arise between valid transactions created simultaneously by honest participants,
+requiring more sophisticated resolution mechanisms.
+
+This simplified approach enables deterministic conflict resolution through
+transaction execution bitmaps and rolling window corrections, avoiding complex
+distributed coordination problems that could delay deployment significantly. The
+protocol also maintains familiar transaction semantics, deterministic ordering,
+and predictable finality patterns that existing dApps and infrastructure depend
+on today.
+
+<a name="downstream-impact"></a>**3. Minimal downstream impact: Ecosystem
+preservation**
+
+Beyond preserving transaction behavior, the design minimizes infrastructure and
+operational disruption for the existing ecosystem. The proposed protocol still
+functions as an overlay extending Praos - like the research paper version,
+allowing SPOs to upgrade progressively without coordinated migrations.
+
+The most obvious approach to increasing throughput while minimizing disruption
+would be increasing Praos block sizes. However, this naive alternative would
+create proportionally longer propagation times that violate Praos timing
+assumptions and lack sufficient scalability for long-term viability.
+Additionally, Praos blocks larger than approximately 3 MB would pose security
+risks by increasing the frequency of short forks that adversaries could exploit
+to compromise the common prefix property and enable attacks such as
+double-spending.
+
+<a name="competitiveness"></a>**4. Competitive positioning**
+
+The coupled block production design can be extended towards higher concurrency
+models, as demonstrated in simulation results. It maintains compatibility with
+more aggressive scaling approaches including full Leios variants, EB and IB
+(input block) decoupling, and sharding extensions, ensuring current throughput
+gains do not preclude 100x+ improvements when chain growth solutions mature.
```

Comment:
Could: mention that we also decided against them because of complexity (above)

```suggestion
gains do not preclude 100x+ improvements when chain growth solutions mature and once the ecosystem is ready to tackle the complexity coming with it.
```